src/Microsoft.SqlTools.ServiceLayer/Security/SecurityService.cs

```diff
@@ -317,7 +317,7 @@ await requestContext.SendResult(new LoginViewInfo()
                 SupportWindowsAuthentication = prototype.WindowsAuthSupported,
                 SupportAADAuthentication = prototype.AADAuthSupported,
                 SupportSQLAuthentication = true, // SQL Auth support for login, not necessarily mean SQL Auth support for CONNECT etc.
-                CanEditLockedOutState = true,
+                CanEditLockedOutState = parameters.IsNewObject,
```

Comment:
I don't think this is how SSMS determines the state. for new login, it is definitely false (I am not even showing it), for existing login, it is only enabled when the login is locked out and current user has permission to unlock the login.

Reply:
for March release, I think I'd simply hide this property to avoid confusion and you can take the time to fix it in May release. sounds good?

Reply:
We can probably test again with this PR merged and if it's still not correct we can hide this.